sam_bot_description/config/bridge_config.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- ros_topic_name: "/clock"
+  gz_topic_name: "/clock"
+  ros_type_name: "rosgraph_msgs/msg/Clock"
+  gz_type_name: "gz.msgs.Clock"
+  direction: GZ_TO_ROS
+
+- ros_topic_name: "/demo/imu"
+  gz_topic_name: "/demo/imu"
+  ros_type_name: "sensor_msgs/msg/Imu"
+  gz_type_name: "gz.msgs.IMU"
+  direction: GZ_TO_ROS
+
+# Topic published by DiffDrive plugin
+- ros_topic_name: "/demo/odom"
+  gz_topic_name: "/demo/odom"
+  ros_type_name: "nav_msgs/msg/Odometry"
+  gz_type_name: "gz.msgs.Odometry"
+  direction: GZ_TO_ROS
+
+# Topic published by JointStatePublisher plugin
+- ros_topic_name: "/joint_states"
+  gz_topic_name: "/joint_states"
+  ros_type_name: "sensor_msgs/msg/JointState"
+  gz_type_name: "gz.msgs.Model"
+  direction: GZ_TO_ROS
+
+# Topic subscribed to by DiffDrive plugin
+- ros_topic_name: "/demo/cmd_vel"
+  gz_topic_name: "/demo/cmd_vel"
+  ros_type_name: "geometry_msgs/msg/TwistStamped"
+  gz_type_name: "gz.msgs.Twist"
+  direction: ROS_TO_GZ
+
+- ros_topic_name: "/scan"
+  gz_topic_name: "/scan"
+  ros_type_name: "sensor_msgs/msg/LaserScan"
+  gz_type_name: "gz.msgs.LaserScan"
+  direction: GZ_TO_ROS
+
+- ros_topic_name: "/scan/points"
+  gz_topic_name: "/scan/points"
+  ros_type_name: "sensor_msgs/msg/PointCloud2"
+  gz_type_name: "gz.msgs.PointCloudPacked"
+  direction: GZ_TO_ROS
+
+- topic_name: "/depth_camera/image"
```

Comment:
There is an image bridge, maybe would be good to use that? https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_image I have some examples doing this in some AMD demos https://github.com/open-navigation/opennav_amd_demonstrations/blob/main/honeybee_gazebo/launch/simulation.launch.py#L124-L144. It uses image transport which gives you some more options like using compression

Reply:
Done in [29276cb](https://github.com/ros-navigation/navigation2_tutorials/pull/102/commits/29276cbcfe4fdf6c89f9f7b312f9e39b67dd0b64)